sdk/python/setup.py

```diff
@@ -86,6 +86,10 @@
     "docker>=5.0.2",
 ]
 
+DQM_REQUIRED = [
```

Comment:
if we support other libraries like GE, do we plan to have multiple "extras_require" ?

If so, I would suggest to name that one "GE_REQUIRED". `feast[greatexpectations]` or `feast[ge]` looks more explicit while DQM is the too broad if we support multiple profilers.